api/src/main/java/jakarta/data/metamodel/Attribute.java

```diff
@@ -127,7 +127,7 @@ public interface Attribute<T> {
      * @return the declaring class
      * @throws UnsupportedOperationException if the declaring type is not
      *                                       known.
-     * @apiNote This is only guaranteed to be known if a static <code>of</code>
+     * apiNote This is only guaranteed to be known if a static <code>of</code>
```

Comment:
Will this cause the text `apiNote` to appear in the Javadoc?  Same comment in 2 other places.

Reply:
Yes, the API note will appear in javadoc the way it is right now.
Since `@apiNote` is not a supported javadoc tag it was not clear to me what the purpose is. 
Do we want these notes in javadoc? Or are these notes only for implementations of Data?
Based on that answer I can fix these notes (as they are in the incorrect place the way they are now).

Reply:
If I remember correctly, the usage of `@apiNote` was asked for by a reviewer at one point.  I just checked and it is not showing in the Javadoc, but does appear in some IDE's,

<img width="471" height="186" alt="image" src="https://github.com/user-attachments/assets/8856c54d-5bea-4ff7-9d7b-a20e628f909e" />